### ISSUE TITLE
inference speed performance improvement

### ISSUE THREAD

**Issue by w19787** [keithito/tacotron#152]
the model will take 15s-20s to synthesize a sentence running a 2.8Ghz CPU with 8GB memory. it is not practical to apply the model on embedded device which don't have cloud support.

how can we speed up the inference speed? any suggestion? thanks

**Comment by rafaelvalle** [comment]
Run it on a GPU? Jetson TK1?

**Comment by w19787** [comment]
@rafaelvalle no, i intend to run the model on android embedded system, which has no GPU support for the deep learning. only CPU supported. so i am wondering how to speed up.

**Comment by superhg2012** [comment]
use tacotron2 instead of tacotron1 or synthesize sound on server cloud

**Comment by rafaelvalle** [comment]
assuming you want high quality audio, take a look at wavernn.
authors claim it runs on a cell phone...

**Comment by rafaelvalle** [comment]
Although there are no audios, you can use MOS (Mean Opinion Score) as a relative proxy to how good it sounds using Wavenet's MOS as a reference.
